tgui/packages/tgui/interfaces/RoboticsControl.tsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author FlameArrow57 (https://github.com/FlameArrow57)
+ * @license ISC
+ */
+
+import { Button, NoticeBox, Section, Table } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface GPSInfo {
+  user_is_ai: boolean;
+  user_is_cyborg: boolean;
```

Comment:
```
  user_is_ai: BooleanLike;
  user_is_cyborg: BooleanLike;
```
(With relevant import)

Reply:
What's the difference between boolean and BooleanLike?

Reply:
boolean is a built-in TypeScript type for `true`/`false`. When serializing data in `ui_data`, because BYOND represents booleans as `1` and `0` (we have macros for `TRUE`/`FALSE`) they get serialized as `1` and `0`. a BooleanLike is a tgui thing (see any other usage) that hints that it shouldn't be treated as a boolean for some special cases. A common one is when rendering, e.g. `{isFoo && <Whatever />}`, if `isFoo` is `0` then this will short-circuit to rendering `0` instead of nothing. That's why we do a lot of `!!` in render code, to convert BooleanLike to actual booleans (there are other ways, `!!` is just a very compact way of doing so).